site/layouts/shortcodes/orange-footer.html

```diff
@@ -0,0 +1,181 @@
+{{- /*
+  Usage: `orange-footer args`
+
+  `args` are all optional and can be one of the following:
+    * mailing: Display mailing content - default = false
```

Comment:
Shall we keep this wording `mailing` to refer to the upper footer's part ? This wording matches the design example but not this boosted footer documentation where this part is called `footer-title-content` to let know that it can contain something else that a title and a mail form, such as a title and and any kind of content.
If we replace `mailing` by something else like `title-content` or `footer-title-content` (or something else to be define), it should be done everywhere, including the variables beginning with `$footer-mailing-`.